charts/wharf-helm/README.md

```diff
@@ -77,42 +77,63 @@ helm install my-release iver-wharf/wharf-helm
 
 ### `api.db.driver`
 
-> Currently only `"postgres"` is a valid value here. Set to `null` or empty string `""` if you wish to populate the `DBHOST`, `DBPORT`, `DBUSER`, `DBPASS` environment variables yourself via `api.extraEnvs` Sets `DBDRIVER` environment variable. Does not support ["Smart environment fields"](./README.md#smart-environment-fields)
+> Currently only `"postgres"` is a valid value here. Set to `null` or empty string `""` if you wish to populate the `WHARF_DB_HOST`, `WHARF_DB_PORT`, `WHARF_DB_USER`, `WHARF_DB_PASS` environment variables yourself via `api.extraEnvs`. Sets `WHARF_DB_DRIVER` environment variable. Does not support ["Smart environment fields"](./README.md#smart-environment-fields)
 
 *Type:* `string`\
 *Default:* `"postgres"`
 
 ### `api.db.host`
 
-> Database hostname used when connecting to the database. Sets `DBHOST` environment variable. Supports ["Smart environment fields"](./README.md#smart-environment-fields)
+> Database hostname used when connecting to the database. Sets `WHARF_DB_HOST` environment variable. Supports ["Smart environment fields"](./README.md#smart-environment-fields)
 
 *Type:* `string`\
 *Default:* `"wharf-db"`
 
+### `api.db.maxConnLifetime`
+
+> Maximum age for a given database connection. If any connection exceeds this limit, while not it use, it will be disconnected. Sets `WHARF_DB_MAXCONNLIFETIME` environment variable. Value is a Go duration, see [Go docs](https://pkg.go.dev/time#ParseDuration). Supports ["Smart environment fields"](./README.md#smart-environment-fields)
```

Comment:
typo
```suggestion
> Maximum age for a given database connection. If any connection exceeds this limit, while not in use, it will be disconnected. Sets `WHARF_DB_MAXCONNLIFETIME` environment variable. Value is a Go duration, see [Go docs](https://pkg.go.dev/time#ParseDuration). Supports ["Smart environment fields"](./README.md#smart-environment-fields)
```